internal/client/model.go

```diff
@@ -1562,16 +1566,18 @@ type CreateIdentityKubernetesAuthResponse struct {
 
 type UpdateIdentityKubernetesAuthRequest struct {
 	IdentityID              string                         `json:"identityId"`
-	KubernetesHost          string                         `json:"kubernetesHost"`
-	CACERT                  string                         `json:"caCert"`
-	TokenReviewerJwt        string                         `json:"tokenReviewerJwt"`
+	KubernetesHost          *string                        `json:"kubernetesHost"`
+	CACERT                  string                         `json:"caCert,omitempty"`
+	TokenReviewerJwt        string                         `json:"tokenReviewerJwt,omitempty"`
```

Comment:
by making it omitempty doesn't this mean that you wouldn't be able to unset these since it's for the PATCH request?